src/armlsd.py

```diff
@@ -115,6 +116,8 @@ def ARMLSD(argflag, spect, fitsdict, reuseMaster=False):
             slf.SetFrame(slf._rordpix, rordpix, det)
             # Save QA for slit traces
             arqa.slit_trace_qa(slf, slf._mstrace[det-1], slf._lordpix[det-1], slf._rordpix[det-1], extord, desc="Trace of the slit edges")
+#            slf._qa.close()
```

Comment:
Remove this and the other slf._qa.close() lines that follow.
We may want, however, to add a try/except clause that
catches a failed run and closes the PDF on a failure.